proxy/envoy/mixer.go

```diff
@@ -101,6 +104,11 @@ func mixerHTTPRouteConfig(role proxy.Node) *FilterMixerConfig {
 		},
 		QuotaName: MixerRequestCount,
 	}
+	if len(service) > 0 {
+		r.MixerAttributes[AttrTargetService] = service
+	}
+
+	return r
```

Comment:
why can't this go into the MixerAttributes struct itself? I believe that struct has been declared with `omit_empty` thingy..

Reply:
I'm not entirely sure if setting a value to "" will cause it to be sent as empty value, or how omit_empty may interact.
This seems more readable and explicit.